tests/src/sanity.rs

```diff
@@ -965,4 +965,16 @@ mod tests {
         assert!(res.success());
         process.assert_stderr();
     }
+
+    #[rstest]
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    pub async fn test_go_outgoing_traffic_single_request_enabled(#[future] service: EchoService) {
+        let service = service.await;
+        let command = vec!["go-e2e-outgoing/go-e2e-outgoing"];
```

Comment:
add 18/19 versions (see PR that's about to be merged #365

Reply:
Added.